.circleci/config.yml

```diff
@@ -75,10 +75,10 @@ jobs:
 
       - store_artifacts:
           path: dist/katex.js
-          destination: katex.js
+          destination: katex.min.js
```

Comment:
The sources and destinations have mismatching filenames.  Did you mean `dist/katex.min.js`?  Ditto below

Reply:
@edemaine Yes, thank you!